roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,19 @@
+- job:
+    name: 'dib-build'
+    node: 'ubuntu-xenial'
+    builders:
+      shell: |
+        virtualenv $HOME/dib-build
+        . $HOME/dib-build/bin/activate
+        pip install diskimage-builder
+
+        git clone https://github.com/BonnyCI/hoist.git /opt/hoist
+        export ELEMENTS_PATH=/opt/hoist/roles/nodepool/files/etc/nodepool/elements/nodepool
+        disk-image-create -x -t qcow2 --checksum --no-tmpfs     \
+                          --qemu-img-options 'compat=0.10'      \
+                          -o /opt/nodepool/images/ubuntu-xenial \
+                          ubuntu-minimal vm
```

Comment:
We need to specify the elements from the hoist repo that we want to include in the build and test.